### ISSUE TITLE
my val loss is't decrese from 23 

### ISSUE THREAD

**Issue by mzeynali** [qjadud1994/CRNN-Keras#19]
Hi @qjadud1994 
I trained your Plates with your setting training but the loss is't reach that you have achieve 0.01 , but my val loss be 26 , why ?

OS : Ubuntu 16.04 LTS
GTX : 1080 ti
RAM :32 GB

**Comment by qianjinhao** [comment]
I have the same problem and don't know wht

**Comment by mzeynali** [comment]
I tested this problem, i've solved this problem with number of large dataset. around 500k images got the val_loss 0.12.

**Comment by allanruin** [comment]
@mzeynali hi, how long does it take you to get val_loss to 0.12, I try a dataset which contains around 700k images, and use Nvidia M40 , at least the first 9 hours, model's val_loss stay around 17. I try to load the checkpoint model weights and see the prediction, it is garbage, something like '.8********', and all the prediction is the same value. Have you encounter this problem?  Could you share some tips, thank you!
